y2m

```diff
@@ -138,7 +138,7 @@ do_clone() # {{{
   fi
   # clean shallow copy
   echo "Unshallowing git repo $moddir..."
-  cd $moddir && { git pull -q --unshallow; cd -; }
+  cd $moddir && { git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"; git pull -q --unshallow; cd -; }
```

Comment:
Wouldn't it be better to prevent the problem in the first place, by using `git clone --depth=1 --no-single-branch ...` above?
Anyway, the line is too long.